build/docker/client/Dockerfile

```diff
@@ -1,13 +1,12 @@
-FROM redhat/ubi9-minimal as users
+FROM redhat/ubi9-minimal
 
 RUN microdnf install shadow-utils -y
 
 RUN groupadd -g 1002 pmm-agent && \
     useradd -u 1002 -r -g pmm-agent -s /sbin/nologin \
-            -d /usr/local/percona/pmm2 \
-            -c "PMM 2.X Client User" pmm-agent
+    -d /usr/local/percona/pmm2 \
+    -c "PMM 2.X Client User" pmm-agent
```

Comment:
Can you please indent `-d` and -c switches under `-u`. So that it becomes clear that they are part of `useradd` command just for readability.

Reply:
Done